src/hotspot/share/gc/shenandoah/shenandoahHeap.hpp

```diff
@@ -506,6 +506,11 @@ class ShenandoahHeap : public CollectedHeap {
   void sync_pinned_region_status();
   void assert_pinned_region_status() NOT_DEBUG_RETURN;
 
+// ---------- Concurrent Stack Processing support
+//
+public:
+  bool uses_stack_watermark_barrier() const override { return true; }
```

Comment:
I wonder that this single use of `override` caused CLang to complain about the rest of the file: https://github.com/zhengyu123/jdk/runs/1762873416